test/lib/mayaUsd/fileio/testImportChaser.py

```diff
@@ -55,6 +58,18 @@ def PostImport(self, returnPredicate, stage, dagPaths, sdfPaths, jobArgs):
 
         print("Info from import:\n" + sdfPathsStr + stageTraverseStr + customLayerDataStr)
 
+        # Get the Sdf path to Dag path matching of all imported prims
+        # Call GetSdfToDagMap() to get the sdfToDagMap, 
+        # which contains SDF path to Dag path mapping
+        sdfToDagMap = self.GetSdfToDagMap()
+        assert(primCount == len(sdfToDagMap))
+        sdfToDagMappingStr = "Sdf to Dag mapping: \n"
+        for sdfToDag in sdfToDagMap:
+            sdfToDagMappingStr += (str(sdfToDag.key()) + "\n")
+            sdfToDagMappingStr += (str(sdfToDag.data()) + "\n")
+
```

Comment:
One more thing you should do here is to verify the content of sdfToDagMap like: 
self.assertEqual(sdfToDagMap[0].key(), "WhateverKeyExpect");
self.assertEqual(sdfToDagMap[0].data(), "WhateverDataExpect");
....

Reply:
Thanks for the suggestion!